Source/ACE.Database/Models/World/TreasureDeath.cs

```diff
@@ -23,5 +23,29 @@ public partial class TreasureDeath
         public int MundaneItemMaxAmount { get; set; }
         public int MundaneItemTypeSelectionChances { get; set; }
         public DateTime LastModified { get; set; }
+
```

Comment:
going to need to find a different way to do this part, scaffolding will blow this out every time

Reply:
yeah my bad, i completely forgot this was a scaffolding generated file

this method should be easily moveable to a static method in another class, or an extension method: deathTresure.Clone()

Reply:
Removed